src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,56 @@
+.stopwatch {
+  position: relative;
+  width: $stopwatch-size;
+  height: $stopwatch-size;
+
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  &__minutes-hand {
+    position: absolute;
+    bottom: 50%;
+    left: center-stopwatch-elements($minutes-hand-width);
+
+    height: $minutes-hand-hight;
+    width: $minutes-hand-width;
+
+    background-color: #0700ff;
+
+    transform-origin: bottom;
+    animation: hand-move 3600s steps(60) infinite;
+  }
+
+  &__seconds-hand {
+    position: absolute;
+    bottom: 50%;
+    left: center-stopwatch-elements($seconds-hand-width);
+
+    height: $seconds-hand-hight;
```

Comment:
There seems to be a typo in the variable name '$seconds-hand-hight'. It should be '$seconds-hand-height' according to the standard naming convention.